config/src/invariant.rs

```diff
@@ -61,6 +64,7 @@ impl InlineConfigParser for InvariantConfig {
                 "depth" => conf_clone.depth = parse_config_u32(key, value)?,
                 "fail-on-revert" => conf_clone.fail_on_revert = parse_config_bool(key, value)?,
                 "call-override" => conf_clone.call_override = parse_config_bool(key, value)?,
+                "shrink-sequence" => conf_clone.shrink_sequence = parse_config_bool(key, value)?,
```

Comment:
added the new `shrink-sequence` key to the invariant inlineconfigparser so it works with #4744